src/clang-c-frontend/clang_c_language.cpp

```diff
@@ -277,7 +277,9 @@ bool clang_c_languaget::preprocess(
 
 bool clang_c_languaget::final(contextt &context, const messaget &msg)
 {
+  // populate symbol value using c2goto operational models
```

Comment:
I don't understand this comment: which symbol value gets populated?

Reply:
Done. Removed the comment.